flask_of_oil/jwt_validator.py

```diff
@@ -133,4 +150,5 @@ def load_keys(self):
         # load the jwk set.
         jwks = KEYS()
         jwks.load_jwks(self.get_jwks_data())
+        self.lastSuccessfulJWKSFetch = datetime.now().timestamp()
```

Comment:
`AttributeError: 'datetime.datetime' object has no attribute 'timestamp'`
are you sure that works?

Reply:
nvm that was python2. works in python3